src/app/pages/blog/blog.js

```diff
@@ -3,61 +3,161 @@ import settings from 'settings';
 import {on} from '~/helpers/controller/decorators';
 import routerBus from '~/helpers/router-bus';
 import analytics from '~/helpers/analytics';
-import Article from './article/article';
-import {description as template} from './blog.html';
 import css from './blog.css';
+import SearchBar from './search-bar/search-bar';
+import SearchResults from './search-results/search-results';
+import UpdateBox from './update-box/update-box';
+import FeaturedArticle from './pinned-article/pinned-article';
+import MoreStories from './more-stories/more-stories';
+import {blurbModel} from './article-summary/article-summary';
+import Article from './article/article';
+import DisqusForm from './disqus-form/disqus-form';
 
+function slugsSortedByArticleDate(articles) {
+    return Object.keys(articles)
+        .sort((a, b) => {
```

Comment:
Can this be reduced to `return articles[a].date < articles[b].date ? 1 : -1;`

Reply:
It can. I'll do that.